Feat/hulu nodefeat: added test script for Node-auth part

justinmusti opened the pull request:
Setup client packge.

Setup uppy package.

Setup local Vite dev server for testing.

filelib commented on .circleci/config.yml:
Place this file as a workflow file into `.github/workflows` since this will be ran by the GitHub Actions

HuluWZ committed:
feat: implement filelib-npm for node and browser platforms
- Add client package with browser and node implementations
- Implement authentication, file reading, and upload functionality
- Add comprehensive test suite for browser and node environments
- Set up GitHub Actions workflow for CI/CD
- Add examples and documentation
- Configure ESLint, Prettier, and TypeScript
